include/wlr/interfaces/wlr_output.h

```diff
@@ -26,14 +26,11 @@ struct wlr_output_impl {
 };
 
 void wlr_output_init(struct wlr_output *output, struct wlr_backend *backend,
-	const struct wlr_output_impl *impl);
-void wlr_output_free(struct wlr_output *output);
```

Comment:
No wlr_output_destroy or similar to replace that?

Reply:
There's `wlr_output_destroy` in `include/wlr/types/wlr_output.h`. `wlr_output_free` was not implemented.